packages/jest-jasmine2/src/index.js

```diff
@@ -54,10 +54,7 @@ function jasmine2(
   let env;
   let jasmine;
 
-  const reporter = new JasmineReporter({
-    noHighlight: config.noHighlight,
-    noStackTrace: config.noStackTrace,
```

Comment:
These config options aren't used in the JasmineReporter or JasmineFormatter. JasmineFormatter does however look at another config option so it seems better to allow this to be a straight passthrough.